__tests__/mock-firestore.test.js

```diff
@@ -14,6 +14,13 @@ describe('Queries', () => {
         name: 'Homer',
         occupation: 'technician',
         address: { street: '742 Evergreen Terrace' },
+        // May 12, 1956.  Conveniently, a negative number
+        birthdate: {
```

Comment:
Is this the only structure for dates that firestore will allow? Will it convert `birthdate: new Date()`?

Reply:
I genuinely do not know.  I would suspect not.  I'm not sure how the roundtrip would work, but I think if FS did accept JS dates, on the backend it would convert them to `timestamp`, and on fetch you receive a `Timestamp` back.

I don't recall why this is in the test.  I think it might be better to remove it, as it doesn't seem to have any utility.

Apologies, I wrote this quite a few months ago (was just waiting on the set functionality to setup the PR)